src/controllers/route.js

```diff
@@ -0,0 +1,18 @@
+const express = require('express');
+const path = require('path');
+const router = express.Router();
+
+const home = require('../views/home');
+
+router.get('/', (req, res) => {
+   res.render('home', { activePage: {home: true}});
+});
+
```

Comment:
Thanks for the explanation. We certainly didn't have an idea what the second argument does.